_harp/academy/_about.md

```diff
@@ -2,4 +2,18 @@
 
 <img src="/assets/fac-logo.png" class="fac-logo">
 
-Founders & Coders runs an intensive full-time, 16-week course in full-stack software development using JavaScript and Node.js. Application is highly-selective and we take just 16 students per cohort. After graduation, we expect all of our students to help us deliver the course to the next cohort and to contribute to our various community projects.
+Founders & Coders has created a new model for education.
+
+We run a full-time, 16-week course in full-stack software development using JavaScript and Node.js. 46/47 of our recent graduates are working as software developers and around 40% of our graduates are female.
+
+We aim to give students the best possible training in software development. In return we ask you to help us deliver subsequent courses and to contribute to our various community projects.
+
+We also ask you to contribute to the academy's sustainability in one of three ways:
+
+- Join our community - earning a living while learning and working on interesting projects with our senior developers.
+
+- Let us help you find full time employment with one of our hiring partners who will provide us with a referral fee.
+
+- Donate 10% of your salary for your first year of employment.
```

Comment:
Is this going to be a blanket 10% or up to a certain amount? Very wary that people paying the new student fees will have a ton of debt already being taken out of their pay.